en/intro/conventions.rst

```diff
@@ -7,10 +7,19 @@ conventions, you get free functionality, and you liberate yourself from the
 maintenance nightmare of tracking config files. Conventions also make for a very
 uniform development experience, allowing other developers to jump in and help.
 
+.. note::
+
+    CamelCase is the practice of writing compound words or phrases so that each word or abbreviation in the middle of the sentence begins with an uppercase letter.
```

Comment:
"in the middle" no, every word

- CamelCase
- camelBacked

see Inflector and other sources here.